config/main.py

```diff
@@ -519,7 +519,100 @@ def is_ipaddress(val):
         return False
     return True
 
+# class for locking entire config process
+class ConfigDbLock():
+    def __init__(self):
+        self.lockName = "LOCK|configDbLock"
+        self.timeout = 10
+        self.pid = os.getpid()
+        self.client = None
+
+        self._acquireLock()
+        return
+
+    def _acquireLock(self):
+        try:
+            # connect to db
+            db_kwargs = dict()
+            configdb = ConfigDBConnector(**db_kwargs)
+            configdb.connect()
+
+            self.client = configdb.get_redis_client('CONFIG_DB')
+            # Set lock and expire time. Process may get killed b/w set lock and
+            # expire call.
+            if self.client.hsetnx(self.lockName, "PID", self.pid):
+                self.client.expire(self.lockName, self.timeout)
+                print(":::Lock Acquired:::")
```

Comment:
Replace `print` statements with `click.echo` statements for consistency.

Also, I feel like we don't need to output success messages, only failure messages. Do you agree?

Reply:
@jleveque : sure I will replace print statements with click.echo.

Network operation folks may have few scripts which uses "config <>" commands. I expect few failures there, once this change is merged (in case, there are few scripts running simultaneously). So yeah, failure case information is useful.

But I can convert successful cases to debug messages, they were more or less for testing. I agree with you.

Reply:
resolved.